schema/patient.js

```diff
@@ -14,6 +14,11 @@ const patientSchema = `
     DELETED
   }
 
+  enum gender {
```

Comment:
since `Gender` is a type/enum, we should capitalize it as a convention (may also affect frontend schema types)
```suggestion
  enum Gender {
```